packages/gestalt/src/SearchField.js

```diff
@@ -10,12 +10,15 @@ import formElement from './FormElement.css';
 import Box from './Box.js';
 import Icon from './Icon.js';
 import FormErrorMessage from './FormErrorMessage.js';
+import FormLabel from './FormLabel.js';
 import { type AbstractEventHandler } from './AbstractEventHandler.js';
 
 type Props = {|
   accessibilityLabel: string,
+  accessibilityClearButtonLabel?: string,
```

Comment:
I made this optional for now, but can go back later and make it required and do a codemod

Reply:
Might make sense to do at the same time as Typeahead